dss/api/search.py

```diff
@@ -99,24 +99,23 @@ def _es_search_page(es_query: dict,
         es_query['_source'] = False
 
     # https://www.elastic.co/guide/en/elasticsearch/reference/5.5/search-request-search-after.html
-    sort = [
-        "manifest.version:desc",
-        "uuid:desc"]
+    es_query['sort'] = [
+        {"uuid": {"order": "desc"}},
+        {"manifest.version": {"missing": "last", "order": "desc"}}
```

Comment:
Just so I can catch up on the context, can you explain the consequences of "missing"?

Reply:
if the field is missing the value will be "last" by default. The value of last will put it at the end of search results. This default value is not stored and only used for determining sort order, and for determining the point to search after.